tests/e2e/test_pgsql.py

```diff
@@ -78,19 +79,24 @@ def test_sql_workload_simple(self, ripsaw):
         log.info(f"Waiting {wait_time} seconds...")
         time.sleep(wait_time)
 
+        # Get pg_bench pod name
+        jsonpath = '{.items[0].status.uuid}'
+        uuid = run_cmd(f"bin/oc get benchmarks -o jsonpath='{jsonpath}'")
+        uuid = uuid.split('-')[0]
         pgbench_pod = run_cmd(
-            'bin/oc get pods -l '
-            'app=pgbench-client -o name'
+            f'oc get pods -l app=pgbench-client-{uuid} -o name'
```

Comment:
bin/oc?

Reply:
Removed oc command from the test, using 'wait_for_resource' instead.